### ISSUE TITLE
Handling interests whose ID starts with a number

### ISSUE THREAD

**Issue by nikkithecrazyazn** [shahriarhossain/MailChimp.Api.Net#22]
I am trying to incorporate "Interests" from your code; however, the IDs in mailchimp start with a number, rather than a latter, resulting in compiler errors.

If a Mailchimp InterestID is, for instance, 555555dddddd, how can we update a member with that interest?

**Comment by shahriarhossain** [comment]
@nikkithecrazyazn  : please paste your code, this will be easier for me to debug the issue.

**Comment by nikkithecrazyazn** [comment]
public class Interests
{
public bool a4889a3400 { get; set; }  
public bool f05129c218 { get; set; }
public bool 556345532c { get; set; } 
public bool 743a843298 { get; set; } 
public bool 86919206a0 { get; set; }
}

I cannot compile because of the interests with IDs of 556345532c and the others that start with numbers.

"Invalid token 556345532c in class, struct or interface member declaration."

**Comment by nikkithecrazyazn** [comment]
I was able to resolve this.

I changed interests to a Dictionary<string, bool> in the MCMember.cs class.
In the code, when I needed to set an interest:

List<Dictionary<string, bool>> listInterest = new List<Dictionary<string, bool>>();
listInterest.Add("f05129c218", true);
listInterest.Add("556345532c", false);

MCMember member = new MCMember();
member.interests = listInterest.

This worked perfectly for interests whose IDs started with numbers.
Didn't need to use the Interests class at all.
